toqito/channel_metrics/completely_bounded_trace_norm.py

```diff
@@ -12,23 +12,15 @@ def completely_bounded_trace_norm(phi: np.ndarray) -> float:
     Find the completely bounded trace norm of a quantum channel.
 
     Also known as the completely bounded diamond norm of a quantum
-    channel [WatCNorm18]_. The algorithm in p.11 of [WatSDP09] with
-    implementation in QETLAB [JohQET] is used.
+    channel (Section 3.3.2 of :cite:`watrous_2018`). The algorithm in p.11 of :cite:`watrous2012simpler` with
```

Comment:
yes, I created a new issue for this #276 . I was lazy and mostly used what was provided in the bib info by arxiv because the changes are becoming too big to keep track of in this PR.